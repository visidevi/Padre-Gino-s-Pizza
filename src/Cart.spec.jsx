import { expect, test } from "vitest";
import { render } from "@testing-library/react";
import Cart from "./Cart";

test("Cart component renders correctly", () => {
  const { asFragment } = render(<Cart cart={[]} />);
  expect(asFragment()).toMatchSnapshot();
});
