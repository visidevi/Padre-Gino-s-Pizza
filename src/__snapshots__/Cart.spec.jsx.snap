// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Cart component renders correctly 1`] = `
<DocumentFragment>
  <div
    class="cart"
  >
    <h2>
      Cart
    </h2>
    <ul />
    <p>
      Total: $0.00
    </p>
    <button>
      Checkout
    </button>
  </div>
</DocumentFragment>
`;
